### PR TITLE
created a simple product card

### PR DESCRIPTION
**Pull request by Evgenb000**
- [DEMO LINK](https://evgenb000.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://evgenb000.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Comment by BondarenkoDmitriy** [comment]
Unfortunately, there is no way to check, because
https://prnt.sc/ROLEKj-mvQq8

**Comment by Evgenb000** [comment]
> Unfortunately, there is no way to check, because [prnt.sc/ROLEKj-mvQq8](https://prnt.sc/ROLEKj-mvQq8)
i made some changes, its gonna work

File: src/blocks/blocks.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
```
Comment (VinnieJ-2k20) [inline src/blocks/blocks.css:30]: This is default value.

File: src/blocks/stars.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__review-count {
+  font-size: 10px;
+}
```
Comment (VinnieJ-2k20) [inline src/blocks/stars.css:9]: This belongs in the different file.

File: src/blocks/stars.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+.stars__star {
+  background: url(../images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```
Comment (VinnieJ-2k20) [inline src/blocks/stars.css:20]: Need to have all possible selectors here. See checklist.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Some small changes left.
Details in comments.

**Review by danylo-dobrovolskyi — COMMENTED** [review]
everything looks good, but there is some problem with positioning of the stars, let the mentor take a look

**Review by VinnieJ-2k20 — APPROVED** [review]

**Pull request closed without merging**